modules/uabb-star-rating/includes/frontend.php

```diff
@@ -14,8 +14,7 @@
 $rating = ! empty( $settings->rating ) ? $settings->rating : 0;
 $layout = 0;
 ?>
-<div class="uabb-rating-content">
-
+<div class="uabb-rating-content" role="banner" tabindex="0" aria-label="<?php echo esc_attr( $settings->rating ); ?>">
```

Comment:
instead of $settings->rating
use $rating

if any variable is not set it will throw PHP warning. Check for similar instances